### ISSUE TITLE
Documentation for afvalophaling card

### ISSUE THREAD

**Issue by langestefan** [UI-Lovelace-Minimalist/UI#549]
Hi. I would like to use the following card: [Custom-card "Afvalophaling"](https://ui-lovelace-minimalist.github.io/UI/usage/custom_cards/custom_card_afvalophaling/) 

However I can't figure out what formatting I need to have on my garbage sensors. I use afvalbeheer already as required. I currently have it set to YYYY-MM-DD but this doesn't work with the card, as seen below:

![image](https://user-images.githubusercontent.com/37669773/159735301-94988073-a0d1-4868-b2b4-6a5fab102738.png)
 
Can the formatting requirement please be added explicitly to the documentation?

**Comment by Depechie** [comment]
I have similar issues with the card. It currently shows nothing...

Would be nice to know the required config section for the sensor.
Seems the date format should be `%d-$m` as stated here [https://github.com/UI-Lovelace-Minimalist/UI/issues/475#issuecomment-1054195502](https://github.com/UI-Lovelace-Minimalist/UI/issues/475#issuecomment-1054195502)
And the template looks for the word 'Geen' so the config for dutch should be 1.

Things that are not really documented for the moment.

**Comment by Depechie** [comment]
To fix the problem @langestefan shows in his screenshot... Not having a title.
There are 3 variables for the card that are only definied in the NL.yaml file.
So they need to be added to the languages.yaml file too ( somehow adding them to the EN.yaml has no effect... NO IDEA WHY )

I guess the custom card needs some revision.

```
    ulm_ophaling: "Trash collection!"
    ulm_geen_ophaling: "No collection"
    ulm_volgende_ophaling: "Next collection"
```

**Comment by Depechie** [comment]
Maybe @Tommatheussen can help a bit here?

**Comment by Tommatheussen** [comment]
translations for custom cards need to be added into the same folder structure as the code for the custom card like so: 
<img width="262" alt="image" src="https://user-images.githubusercontent.com/13683094/164035884-6a271e46-ec29-407b-90e1-8b92a71e30bc.png">
I agree documentation is lacking a bit, it does look like my latest changes to the documentation for this card have been reverted, I'll need to look into why.
This configuration for the integration works perfectly for me:

```
platform: afvalbeheer
wastecollector: recycleapp
resources:
  - restafval
  - papier
  - pmd
postcode: x
streetnumber: x
streetname: x
upcomingsensor: 1
dateformat: '%-d %B'
dutch: 1
```

**Comment by Depechie** [comment]
Thx @Tommatheussen for some details… I thought I texted the translation file but will try again ( I want to have EN.yaml ).

**Comment by Depechie** [comment]
Ok!
Placing an en.yaml file with following structure now enables a title for the card!
```
---
ulm_language_variables:
  variables:
    ulm_ophaling: "Trash collection!"
    ulm_geen_ophaling: "No collection"
    ulm_volgende_ophaling: "Next collection"
```

**Comment by Alcarecco** [comment]
Hi all, 
I am quite a newbie to programming and Home Assistant, but eager to learn!
Based on the instructions of everythingsmarthome I am trying to build my own dashboard, and also want to have the afvalwijzer custom card. I have the addon from pippyn installed.

I have the same issue as @langestefan
With the instructions above, I have been able to get the following:
![Screenshot_Afvalwijzer](https://user-images.githubusercontent.com/106401913/171990299-52bede30-7c70-492b-81f7-30e1a84bf276.jpg)

However, I cannot get the date before the "GFT" text. 
I am also not able to get the bottom part of the card with future garbage collections to appear.
Any help is very much appreciated!

**Comment by Alcarecco** [comment]
Thanx for your input! I am gonna try some more.
And thanks for the tip on how to create fenced coding blocks. I finally understand :-)

**Comment by TheDescender** [comment]
I might be misunderstanding how the translation files work. When I added this custom card through the built-in configuration menu of Minimalist, it seems the 6 if statements in the custom-card code check the entity state for the collection_today and collection_tomorrow sensors of pippyn's integration using a Dutch string. See below example.

```(variables.ulm_card_ophaling_vandaag && states[variables.ulm_card_ophaling_vandaag].state !== 'Geen')```

This issue results in the future collection functionality of the custom-card not working, since it will never pass these if statements. When the entity state string in the cutom-card code is manually changed from "Geen" to "None", it functions just fine. The custom-card didn't come with any translation file when I downloaded it or at least this seems to have slipped past. Am I missing something that I'm meant to do here or is this just a bug?
